sys-kernel/bootengine/bootengine-9999.ebuild

```
@@ -10,7 +10,7 @@ CROS_WORKON_REPO="https://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="bced9373ee9f11028b3c9d704fbca3d2e059a4ef" # flatcar-master
+	CROS_WORKON_COMMIT="cc2c408c468b126f2b6ff8e04e470880f94fde69" # kai/ignition-v3
```

```suggestion
	CROS_WORKON_COMMIT="506ea2d796ef9ec9e22402763c9d173d7324fe3d" # kai/ignition-v3
```
I have rebased it. This suggestion only has the base directory change, the latest commit tries to port the dracut module but probably won't work now.